### PR TITLE
Add initial console app

### PR DESCRIPTION
**Pull request by indrekj**
screen shot 2017-09-20 at 17 47 08

### CODE REVIEW

**Comment by deiwin** [comment]
Nice! 💻

File: app/app.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    'api.salemove.eu'
+  ].freeze
+
+  def initialize(incoming_url)
+    @incoming_url = incoming_url
+    @prompt = TTY::Prompt.new
+  end
+
+  def run
+    @server      = @prompt.select('SaleMove API location?', API_LOCATIONS)
```
Comment (willmore) [inline app/app.rb:23]: Feature request: read from optional ~/.salemove.yaml

File: app/app.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    @site_id     = ENV['SITE_ID']     || @prompt.ask('Site ID?')
+    @dev_token   = ENV['DEV_TOKEN']   || @prompt.ask('Dev Token?')
+    @operator_id = ENV['OPERATOR_ID'] || @prompt.ask('Operator ID?')
+
+    info ">> Creating a reactive engagement request"
```
Comment (willmore) [inline app/app.rb]: `">> Requesting engagement..."`

File: app/gui/keymaps/_global_.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Vedeu.keymap '_global_' do
+  key(:ctrl_c) { Vedeu.exit }
```
Comment (urmastalimaa) [inline app/gui/keymaps/_global_.rb:2]: Can you document the key bindings in README?

**Review by willmore — CHANGES_REQUESTED** [review]
Please add MIT license.

**Comment by indrekj** [comment]
@willmore updated

**Review by willmore — APPROVED** [review]

**Review by urmastalimaa — APPROVED** [review]

**Comment by indrekj** [comment]
!merge

**Pull request merged**